appveyor.yml

```diff
@@ -0,0 +1,31 @@
+# General Configuration
+version: 0.0.{build}
```

Comment:
What's the relationship between the build "version" here, the AssemblyVersion attributes and the version number in pkg.json?

Reply:
@alvpickmans AssemblyInfo patching could work here for sure!

Reply:
On the other projects I've opted to scrap the `0.0.{build}` version number in favour of a simple `{build}`. It all depends where you want the single source of truth to be 🤷🏻‍♂️